### PR TITLE
Private/gokay/impress notes

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] I have run `make prettier-write` and formatted the code.
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by pedropintosilva — COMMENTED** [review]
I have tested, it works. 

But I might I have found something on mobile

```
 Uncaught TypeError: global.app.definitions is undefined
    logServer http://localhost:9980/browser/e8f30403/global.js:506
    onerror http://localhost:9980/browser/e8f30403/global.js:561
    Async* moz-extension://9f161837-29b8-4bff-9888-87d0f50e89ed/page-script.js:293
    <anonymous> moz-extension://9f161837-29b8-4bff-9888-87d0f50e89ed/page-script.js:294
global.js:506:14
```

```
 Uncaught Error: Permission denied to access property "action"
    eventCallback moz-extension://9f161837-29b8-4bff-9888-87d0f50e89ed/page-script.js:7
    Async* moz-extension://9f161837-29b8-4bff-9888-87d0f50e89ed/page-script.js:293
    <anonymous> moz-extension://9f161837-29b8-4bff-9888-87d0f50e89ed/page-script.js:294
page-script.js:7:13

```

**Review by pedropintosilva — APPROVED** [review]
those errors happen without or changes

**Comment by gokaysatir** [comment]
> those errors happen without your changes

Thanks for the review :) Let me also check them.

File: browser/src/control/Control.Notebookbar.js [hunk 1]
```diff
@@ -375,6 +375,17 @@ L.Control.Notebookbar = L.Control.extend({
 		}
 	},
 
+	ignoreContextChange(contextes) {
```
Comment (eszkadev) [inline browser/src/control/Control.Notebookbar.js:378]: please consider renaming to:
shouldIgnoreContextChange - as it returns boolean
ignoreContextChange - sounds like void, "do it"

File: browser/src/docdispatcher.ts [hunk 2]
```diff
@@ -509,6 +509,19 @@ class Dispatcher {
 		this.actionsMap['selectbackground'] = function () {
 			L.DomUtil.get('selectbackground').click();
 		};
+
+		this.actionsMap['notesmode'] = function () {
+			const button = document.getElementById('notesmode');
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/docdispatcher.ts:514]: or maybe we should try to use `var stateHandler = this.map['stateChangeHandler'];` for storing the state of notes mode

then unobutton can fetch that state and apply the selection itself

**Comment by gokaysatir** [comment]
I guess it's safe now to merge this.

**Pull request merged**